README.md

```diff
@@ -106,37 +119,36 @@ pip install pyhf['jax']
 ## Directory structure
 ### source code
 
-- `analysis`: Contains all files related to an analysis. E.g. `ZeroLepton/`
-- `config` : Contains HistFactory schema
+- `analysis`: Contains example configuration files in `tutorial/`.
```

Comment:
This is still where people should put their config files in order to keep the historical workflow.

Reply:
But user config files should not be in the source code analysis folder. I kept that description for the work directory. I will try another wording :)